source/contribute.rst

```diff
@@ -198,6 +247,37 @@ this in case you want to read more about this:
 - `Commit message style for git <https://commit.style/>`_
 - `A note about git commit messages <https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html>`_
 
+Once you are done, you can push your branch to your fork:
+
+.. code-block:: console
+
+    $ git push --set-upstream fork your-sql-support # use -u for short
+
+Creating the pull request
+-------------------------
+
+New pull requests are created with the repository's default branch as
+base branch, and that might not be what you want.
+
+The base branch should be what you chose according to the guidelines
+mentioned in the beginning of this chapter. It is also the branch name
+that you provided when you created your topic branch.
+
+Make sure to pick the correct branch when creating the pull request. If
+you do not, it can still be changed afterwards (see :ref:`the dedicated
+section below<rebasing-on-another-branch>`).
+
+The default branch is the branch you see when you enter the repository
+page on GitHub.
+
+.. image:: ../images/default-branch.png
+   :alt: The default branch
+   :style: margin-bottom: 20px
+
+In this DBAL example, it's the branch with the name **2.11.x**. The
+branch name reflects the current lowest supported version of a
+repository.
```

Comment:
Relevant information?

Reply:
Are you asking if this sentence is relevant, or do you want me to add relevant information?

Reply:
✅